tests/Makefile

```diff
@@ -17,7 +17,9 @@ REPO_ROOT ?= REPO_ROOT=$(ROOT)
 # Nose test runner configuration options.
 NOSECFG = --config .noserc
 
-test: pep8 test_unit test_functional
+test_agnostic: pep8 test_unit
```

Comment:
Repeating should be fine, there are not too many places, and that way it is more visible what is needed for each target.